icons/cog.json

```diff
@@ -10,5 +10,8 @@
   ],
   "categories": [
     "account"
+  ],
+  "contributors": [
```

Comment:
I only added the contributors that are not tracked in git, I can add them by running `pnpm generate:contributors` if you like.

What I don't know how to test is the release workflow.

Reply:
I've added all contributors, I think we definitely need to list everyone, even if it's "redundant" in a way so that people editing icons are not confused.

You can easily test the release workflow, it's just a bit tricky, you have to also update the release script in your own fork (and of course manually create a new release in your fork):
https://github.com/lucide-icons/lucide/blob/main/.github/workflows/release.yml#L31

Reply:
I've moved the target branch to `docs-vitepress`, I think it would make sense for this to go online when the new site does, what do you think @ericfennis & @jguddas?

Reply:
@karsa-mistmere I think It's better to target the main branch, then we can already merge it and test it with the release flow.

Reply:
OK, that also makes sense.